documentation/configuration.md

```diff
@@ -1256,7 +1256,7 @@ A trigger that has its condition satisfied when the number of HTTP requests have
 | Name | Type | Required | Description | Default Value | Min Value | Max Value |
 |---|---|---|---|---|---|---|
 | `RequestCount` | int | true | The threshold of the number of slow requests that start within the sliding window of time. | | | |
-| `RequestDuration` | Timespan? | false | The threshold of the amount of time in which a request is considered to be slow. | `"00:00:05"` (5 seconds) | `"00:00:00"` (zero seconds) | `"01:00:00"` (1 hour) |
+| `RequestDuration `  | TimeSpan? | false | The threshold of the amount of time in which a request is considered to be slow. | `"00:00:05"` (5 seconds) | `"00:00:00"` (zero seconds) | `"01:00:00"` (1 hour) |
```

Comment:
I think this should be reverted.

Reply:
I didn't notice that; yes, that should be fixed. But the extra spaces should be removed.